package.json

```diff
@@ -37,7 +37,14 @@
         "isomorphic-fetch": "^2.2.1",
         "lodash": "^4.17.5"
     },
+    "resolutions": {
+        "babel-core": "^7.0.0-bridge.0"
```

Comment:
Never seen this before. What does this do?

Reply:
this is the link between jest and babel 7, babel-jest uses babel-core but babel-core uses @babel/core now, so this "bridges" that gap

Reply:
Thanks @kaylieEB! 😉

Reply:
For more info, https://github.com/babel/babel-bridge